### PR TITLE
POL-250 - New Resource Table 12

### PR DESCRIPTION
**Pull request by rshade**
### Description

[Describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: operational/stranded_servers/stranded_servers.pt [hunk 1]
```diff
@@ -142,46 +142,62 @@ script "js_get_stranded_instance", type: "javascript" do
...
+  label "Send Mail"
+  description "Sends incident email"
   email $param_email
 end
 
-escalation "approve_terminate_instances" do
-  request_approval  do
+escalation "esc_approve_terminate_instances" do
+  automatic false
   label "Approve Instance Termination"
```
Comment (cdwilhelm) [inline operational/stranded_servers/stranded_servers.pt]: Change to "Delete Instance"

File: operational/stranded_servers/stranded_servers.pt [hunk 2]
```diff
@@ -142,46 +142,62 @@ script "js_get_stranded_instance", type: "javascript" do
...
-    description "Approve escalation to run RightScale Cloud Workflow to delete Stranded Instnace"
-    parameter "approval_reason" do
-      type "string"
-      label "Reason for Approval"
-      description "[Note]: Locked instances will not be terminated."
+  description "Approve escalation to run RightScale Cloud Workflow to delete Stranded Instnace"
```
Comment (cdwilhelm) [inline operational/stranded_servers/stranded_servers.pt]: Change to "Delete selected stranded instances"

File: saas/fsm/suspicious_users/fsm-suspicious_users.pt [hunk 3]
```diff
@@ -142,15 +142,27 @@ policy "policy_fsm_suspicious_users" do
       detail_template <<-EOS
```
Comment (cdwilhelm) [inline saas/fsm/suspicious_users/fsm-suspicious_users.pt]: Needs to bump the version and update the change log

File: saas/fsm/suspicious_users/fsm-suspicious_users.pt [hunk 4]
```diff
@@ -142,15 +142,27 @@ policy "policy_fsm_suspicious_users" do
...
 # Suspicious SaaS Users
```
Comment (cdwilhelm) [inline saas/fsm/suspicious_users/fsm-suspicious_users.pt]: Needs to bump the version and update the change log

File: saas/fsm/unsanctioned_spend/fsm-unsanctioned_spend.pt [hunk 5]
```diff
@@ -239,15 +239,30 @@ policy "policy_fsm_unsanctioned_spend" do
       detail_template <<-EOS
```
Comment (cdwilhelm) [inline saas/fsm/unsanctioned_spend/fsm-unsanctioned_spend.pt]: Needs to bump the version and update the change log

File: saas/fsm/user_status_change/fsm-user_status_change.pt [hunk 6]
```diff
@@ -242,15 +242,27 @@ policy "policy_fsm_inactive_users" do
       detail_template <<-EOS
```
Comment (cdwilhelm) [inline saas/fsm/user_status_change/fsm-user_status_change.pt]: Needs to bump the version and update the change log

File: security/security_groups/high_open_ports/open_ports.pt [hunk 7]
```diff
@@ -127,12 +127,28 @@ policy "rule_open_ports" do
     detail_template <<-EOS
```
Comment (cdwilhelm) [inline security/security_groups/high_open_ports/open_ports.pt]: Needs to bump the version and update the change log

File: security/security_groups/world_open_ports/security_group_rules_with_world_open_ports.pt [hunk 8]
```diff
@@ -168,12 +168,37 @@ policy "ports_open_to_the_world" do
     detail_template <<-EOS
```
Comment (cdwilhelm) [inline security/security_groups/world_open_ports/security_group_rules_with_world_open_ports.pt]: Needs to bump the version and update the change log

File: security/security_groups/rules_without_descriptions/security_group_rules_without_descriptions.pt [hunk 9]
```diff
@@ -123,15 +123,37 @@ policy "rule_without_description" do
     detail_template <<-EOS
```
Comment (cdwilhelm) [inline security/security_groups/rules_without_descriptions/security_group_rules_without_descriptions.pt]: Needs to bump the version and update the change log

File: security/security_groups/icmp_enabled/icmp_enabled.pt [hunk 10]
```diff
@@ -127,16 +127,26 @@ policy "icmp_code" do
     detail_template <<-EOS
```
Comment (cdwilhelm) [inline security/security_groups/icmp_enabled/icmp_enabled.pt]: Needs to bump the version and update the change log

File: operational/stranded_servers/stranded_servers.pt (hunk 2 above)
Comment (SheikSena) [reply operational/stranded_servers/stranded_servers.pt:159]: "Delete selected stranded instances" will be more than 20 characters so I have changed it to "Terminate Instances"

**Comment by SheikSena** [comment]
@cdwilhelm, I have updated all the policies, Except for the following policies:

1. "operational/fnms/schedule_fnms_reports/schedule-fnms-report.pt" It contains only one field [detail_template <<-EOS
{{data.sResult}}
EOS]
2. "operational/fnms/schedule_fnms_reports_on_premise/schedule-fnms-report.pt"
[detail_template <<-EOS
{{data.sResult}}
EOS]
3. "cost/scheduled_reports/scheduled_report.pt" This policy the graphs (optima dashboard data).

Please let me know your thoughts on the above-mentioned policies.

**Comment by cdwilhelm** [comment]
> @cdwilhelm, I have updated all the policies, Except for the following policies:
> 
> 1. "operational/fnms/schedule_fnms_reports/schedule-fnms-report.pt" It contains only one field [detail_template <<-EOS
>    {{data.sResult}}
>    EOS]
> 2. "operational/fnms/schedule_fnms_reports_on_premise/schedule-fnms-report.pt"
>    [detail_template <<-EOS
>    {{data.sResult}}
>    EOS]
> 3. "cost/scheduled_reports/scheduled_report.pt" This policy the graphs (optima dashboard data).
> 
> Please let me know your thoughts on the above-mentioned policies.

@SheikSena   thanks for letting me know about this.  scheduled report was to be excluded from the list.  I missed that.    The FNMS policies are using a dynamically created table.  I didn't realize this.  We'll skip these for now.

**Comment by cdwilhelm** [comment]
@SheikSena   is there any changes left to this PR?

**Comment by SheikSena** [comment]
No @cdwilhelm . Everything is done !

**Pull request merged**